### ISSUE TITLE
Compute FID

### ISSUE THREAD

**Issue by wwwnev** [Olivier-tl/MO-DAGAN#43]
Should compute it separately from generation step.
What to do with results ? Save to file for now ?

**Comment by Olivier-tl** [comment]
Yup I think save to file or print to consol is good 👍🏻. On pourrait compute ça dans un script standalone?
